src/Frontend/Themes/Bootstrap4/src/Layout/Sass/_bootstrap-variables.scss

```diff
@@ -1,7 +1,3 @@
-$fa-font-path: "/fonts/vendors/";
-$fancybox-image-url: '/images/vendors/fancybox/';
-$bootstrap-sass-asset-helper: false;
-
```

Comment:
Heb je gezocht of deze 3 bovenstaande variabelen niet meer gebruikt worden? Ook in de bijhorende node module checken

Reply:
Goed gezien, was me niet opgevallen dat er bovenaan dingen weg waren

Reply:
Heb je gekeken of ze nog nodig zijn? want denk niet dat ze alle drie nog nodig zijn?

Reply:
Jep ik vind ze allemaal terug in node modules